pyproject.toml

```diff
@@ -123,6 +123,8 @@ cohere = { version = "^4.27", optional = true }
 together = { version = "^0.2.8", optional = true }
 weaviate-client = { version = "^3.24.1", optional = true }
 docx2txt = { version = "^0.8", optional = true }
+openpyxl = { version = "^3.1.2", optional = true }
+xlrd = { version = "^2.0.1", optional = true }
```

Comment:
Going forward, we are planning to not add everything to the optional dependencies. Instead, can you please:

1. Add instructions in the docs to install these two packages
2. Try to import these on the loader file and if import fails, show the exception to install these packages.

Reply:
@deshraj I have made the change, please check